app/static/js/index_buttons.js

```diff
@@ -1,7 +1,16 @@
 $(document).ready(function () {
     setPassSaveBtn();
     setDownloadBtns();
-    showXLSXBtn();
+})
+
+$(document).change(function () {
```

Comment:
This function is currently being called on every action. Can you create two separate listener functions (for order_type and status) instead? And those functions can call the existing code below to handle the change event.